pinot-controller/src/main/resources/app/interfaces/types.d.ts

```diff
@@ -28,8 +28,10 @@ declare module 'Models' {
     customRenderer: JSX.Element
   }
 
+  export type MapRecord = Record<string, string | number>
```

Comment:
Record<string, unknown> ?

Reply:
As per the spec the Record `key` is string and `value` is `string | numer`

Reply:
@esakkiraj In the future it would change to other value type as well. Should we make it generic?

Reply:
Suggest to make it generic to accommodate any future change.

Reply:
@raghavyadav01 @xiangfu0 Yes, already updated to generic type.